roles/neutron/vars/main.yml

```diff
@@ -114,6 +120,12 @@ __neutron_ovn_helm_values:
         ml2:
           type_drivers: flat,vlan,geneve
           tenant_network_types: geneve
+      taas:
+        taas:
+          enabled: true
```

Comment:
Since this section gets merged into the original one, having this here is redundant.

The usage of this variable is not one or the other but it starts with the first and overlays this if OVN is in use.